docs/es-modules/subtitles-and-captions.html

```diff
@@ -116,32 +116,32 @@ <h4 class="mt-4 mb-2">Karaoke player</h4>
       import 'cloudinary-video-player/cld-video-player.min.css';
       import 'cloudinary-video-player/playlist';
 
-      const player = videoPlayer('player', {
-        cloudName: 'demo'
+      const player = cloudinary.videoPlayer('player', {
+        cloud_name: 'prod'
```

Comment:
is it expected change?

Reply:
The cloud change?  yes
Adding `cloudinary`? no, but It doesn't really matter

Reply:
I meant mostly about changing `cloudName` to `cloud_name`, isnt `cloudName` expected prop?